static/js/resize.js

```diff
@@ -157,6 +157,7 @@ exports.watch_manual_resize = function (element) {
 
 exports.resize_bottom_whitespace = function (h) {
     $("#bottom_whitespace").height(h.bottom_whitespace_height);
+    $("#compose-textarea").css("max-height", h.bottom_whitespace_height - 78);
```

Comment:
Hey , I haven't mentioned it , but 
![drawisland](https://user-images.githubusercontent.com/53977614/92210505-6f75d400-eeac-11ea-82b1-07acfd6c8c54.png)
The summation of these two region's height ( including padding + margin ) comes upto 78px . 
Hence , maintaining proportionality while resizing the window vertically .

Reply:
Can we replace that hardcoded 78 with a calculation?  That'll be more robust if we end up changing the size of any of these elements.

Reply:
Oh Fine , I wasn't doing this at first was because it would add unnecessary lines of repeating code . Will do it :+1:

Reply:
Oh Cool . The reason I wasn't doing at first was because of unnecessary block of code repeating .Will do it now :+1: